### PR TITLE
Added default values for swagger

### LINKED ISSUE #42: Default template values are not passed to the builder operations
When setting a default value:

``` javascript
{name=defaultValue:type}
```

The default value is not set in the builder

``` javascript
// rest-connector.js line 58
fn.accepts.push({
  arg: p,
  type: args[p].type,
  required: args[p].required,
  http: {source: 'query'}
});
```

This result in swagger not having the data and not setting the input with the default value.

_note: i'm willing to fix and create a test for it_

### PR DESCRIPTION
**Pull request by aaafly**
Fixes #42 Where swagger did not get the default values from the template.
@richardpringle please review.

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by richardpringle** [comment]
ok to test

File: lib/rest-connector.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ exports.initialize = function initializeDataSource(dataSource, callback) {
             fn.accepts.push({
               arg: p,
               type: arg.type,
+              default: arg.default,
```
Comment (raymondfeng) [inline lib/rest-connector.js]: Can we only set `default` if it is not `undefined`?

**Comment by aaafly** [comment]
Yes please, take over.

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by Amir-61** [comment]
Sure I moved it to `Needs Priority` to plan it soon and get it done.

**Pull request closed without merging**